images/Dockerfile

```diff
@@ -47,7 +47,8 @@ RUN apt update -y \
 # Configure git-core/ppa based on guidance here:  https://git-scm.com/download/linux
 RUN add-apt-repository ppa:git-core/ppa \
     && apt update -y \
-    && apt install -y --no-install-recommends git
```

Comment:
install recommends will have `ssh` installed, so customer can check out repository via `ssh` in `actions/checkout`